### PR TITLE
Format SCSS files with prettier

### PR DESCRIPTION
**Pull request by sdepold**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +39,9 @@
     "@tsconfig/docusaurus": "1.0.5",
     "eslint": "8.14.0",
     "eslint-plugin-mdx": "1.17.0",
+    "husky": ">=6",
+    "lint-staged": ">=10",
+    "prettier": "^2.6.2",
```
Comment (ephys) [inline package.json:44]: can you pin these? :)

**Review by ephys — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (sdepold) [reply package.json:44]: Do you know if they will be pinned automatically by renovate later?

Comment (WikiRik) [reply package.json]: I believe so, depending on which default config renovate uses. But either way I think we want to have the same behaviour throughout core/CLI/website

Comment (ephys) [reply package.json]: It will but it prevents an unnecessary PR :)
(don't forget to pin the other two)

Comment (sdepold) [reply package.json]: fixed

**Review by ephys — APPROVED** [review]

**Pull request merged**